### DISCUSSION TITLE
CORE_EXPORT can't be identified.

### DISCUSSION THREAD

**Discussion by pby22555** [NHERI-SimCenter/SimCenterCommon#211] [R2D] [answered]
Dear friends, I’m sorry to disturb you. The CORE_EXPORT can’t be identified in QT, can you tell me what I can do? 
Yours, sincerely. 
Boyan.
![bug](https://github.com/user-attachments/assets/2f8fe6af-5e4d-49e0-af5e-4d89718e6109)

**Comment by pby22555** [comment]
I am developing R2D.pro.
![bug2](https://github.com/user-attachments/assets/1d3e15ed-3be9-40ad-a515-f28f32d31339)

**Reply by fmckenna** [reply, marked as answer]
Have you the QGIS_PLUGIN (https://github.com/NHERI-SimCenter/QGISPlugin) repo? and you are using Qt5.15.2?? 

NOTE: we require 5.15.2 for QGIS.

**Reply by pby22555** [reply]
Thanku, sir

**Comment by pby22555** [comment]
您好，您的邮件我已收到，我会及时查收。谢谢！
